minimal/core_01_minimal.js

```diff
@@ -0,0 +1,258 @@
+/**
+ * @license
+ * Code City: Minimal database.
+ *
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Minimal database for Code City.
+ * @author fraser@google.com (Neil Fraser)
+ */
+
+var user = null;
```

Comment:
This may work well enough for now, but note that both setTimeout and network activity are going to be incompatible with this approach.